src/lib.rs

```diff
@@ -130,6 +130,16 @@ impl Troffable for Roff {
 
         writeln!(&mut res, "{}", self.title.render()).unwrap();
 
+        // Compatibility settings:
```

Comment:
Why is this needed? What requires turning of justification and hyphenation?

I'm looking at this from a manual page generation point of view, and non-justified manual pages are unusual, and thus jarring to read.

Reply:
On the topic of compat settings, asciidoctor does https://github.com/asciidoctor/asciidoctor/blob/main/lib/asciidoctor/converter/manpage.rb#L61 (with links out for information)

Reply:
The point about plain apostrophes is a good one, and I should add that to my roff crate version. However, I can't see any explanation of why hyphenation and justification should be turned off. The sentence space size I can live with: it removes the need to make sure sentences end with a newline after the terminating punctuation.